tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AddWorkspaceToOrganizationTest.java

```diff
@@ -33,7 +33,7 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S})
+@Test(groups = {UNDER_REPAIR})
```

Comment:
You can just add `TestGroup.UNDER_REPAIR` item to the list not removing existed ones.